admin_app/src/app/page.tsx

```diff
@@ -60,14 +61,22 @@ export default function Home() {
         }
       })
       .finally(() => setIsLoading(false));
+    setNewCardText("");
+    setNewCardTitle("");
   };
 
-  const saveNewCardInBackend = (content_text: string) => {
+  const saveNewCardInBackend = (
```

Comment:
Same as before, this should take an object `content_data: ContentData` that encapsulates title and text.